features/svg-animations.yml

```diff
@@ -0,0 +1,97 @@
+name: SVG animations
+description: The `<animate>`, `<animateMotion>`, and `<animateTransform>` SVG elements declaratively animate SVG elements. Also known as SMIL.
+spec: https://svgwg.org/specs/animations/
+caniuse: svg-smil
+group:
+  - animation
+  - svg
+status:
+  compute_from: svg.global_attributes.class.animation_support
```

Comment:
Looking at the dist file, I see 2 main groups: one which became baseline low in 2015, and the other which became baseline low in 2020.
Do you think SMIL was completely usable in 2015? Any idea what the 2020 group added? Is it worth thinking about it in terms of a separate feature? Probably not because it's all baseline high by now.
But if not, then I'd add a comment on this line which explains why we chose this particular compute_from key, even if a lot of SMIL features landed at least 5 years prior to that.

Reply:
It looks like almost all keys are quite old, but didn't have Edge support until Edge adopted Chromium. Perhaps there was earlier Edge support? Might be worth adding to #1499.

Reply:
Oh ok, sounds good then.